dtos/requests/interval_test.go

```diff
@@ -87,7 +87,7 @@ func TestAddIntervalRequest_Validate(t *testing.T) {
 		{"valid AddIntervalRequest", valid, false},
 		{"valid AddIntervalRequest, no Request Id", noReqId, false},
 		{"valid AddIntervalRequest, interval name containing unreserved chars", intervalNameWithUnreservedChars, false},
-		{"invalid AddIntervalRequest, interval name containing reserved chars", intervalNameWithReservedChars, true},
+		{"Valid AddIntervalRequest, interval name containing reserved chars", intervalNameWithReservedChars, false},
```

Comment:
lowercase to meet the original test message

Reply:
Updated.